dir.targets

```diff
@@ -93,4 +93,76 @@
     </ItemGroup>
   </Target>
 
+  <!-- Support for CoreRT Targeting Packs
+       CoreRT Targeting requires differentiating the targetting pack according to the platforms.
+       i.e.: the targetting pack for linux is different than that of windows. In order to support this
```

Comment:
If you do the split targeting pack, it is hard to tell where to stop. Do you also need to split accross CPU architectures? The implementation corelib is different for each x86, x64, arm, arm64. 

I am not saying that it is a good idea to split it accross CPU architectures. I am just pointing out that this plan is not correct by construction, and just makes CoreRT different from CoreCLR.